core/src/main/scala/org/apache/spark/storage/MemoryStore.scala

```diff
@@ -468,11 +474,18 @@ private[spark] class MemoryStore(blockManager: BlockManager, maxMemory: Long)
     }
   }
 
+  def releasePendingUnrollMemoryForThisThread(): Unit = {
```

Comment:
Please add a small java doc here (see other methods in this class for reference).